### PR TITLE
refactor: update module

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
- Use Github Actions instead of CircleCI
- Remove babel
- Increase coverage 100%
- Warn user if any netlify files not found
- Remove dependency `fs-extra`

### CODE REVIEW

**Comment by ricardogobbosouza** [comment]
@manniL what do you think??

**Comment by manniL** [comment]
will have a look tmmrw. thanks for the pr!

File: lib/module.js [hunk 1]
```diff
@@ -1,19 +1,20 @@
 const { resolve } = require('path')
-const fs = require('fs-extra')
+const { copyFileSync, existsSync } = require('fs')
```
Comment (manniL) [inline lib/module.js:2]: I've used `fs-extra` to benefit from parallel file checking. We could theoretically use [`fs.promises`](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api) for that but it won't make a huge difference I guess as we only check for two files 🤷🏻‍♂️

Comment (ricardogobbosouza) [reply lib/module.js:2]: Exactly, it is a simple check.

**Pull request merged**